### DISCUSSION TITLE
Is it possible to get user's firstName and lastName? i only found displayName in the document

### DISCUSSION THREAD

**Discussion by BigSalty** [kreait/firebase-php#472] [Q&A]
**Is your feature request related to a problem? Please describe.**
it seems to be possible to get user's firstName and givenName in JS with `firebaseResult.additionalUserInfo`, but i wonder whether it can be down in PHP

**Comment by jeromegamez** [comment]
Sorry for the late reply! As far as I know, this is a property of the Client SDKs after a successful sign-in, I'm not sure this is possible from the backend side, but I'll have a look!
